exporter/datadogexporter/metadata/host.go

```diff
@@ -66,6 +67,21 @@ func GetHost(logger *zap.Logger, cfg *config.Config) *string {
 	return &hostname
 }
 
+func getClusterName(attrs pdata.AttributeMap) (string, bool) {
+	if k8sClusterName, ok := attrs.Get(conventions.AttributeK8sCluster); ok {
+		return k8sClusterName.StringVal(), true
+	}
+
+	cloudProvider, ok := attrs.Get(conventions.AttributeCloudProvider)
+	if ok && cloudProvider.StringVal() == conventions.AttributeCloudProviderAzure {
+		return azure.ClusterNameFromAttributes(attrs)
+	} else if ok && cloudProvider.StringVal() == conventions.AttributeCloudProviderAWS {
+		return ec2.ClusterNameFromAttributes(attrs)
+	}
```

Comment:
I might not have followed everything here, is there a reason why we're not also doing this for GCP? Is it because there are no ways to get a cluster name from GCP-specific attributes?

Reply:
Yes, I should have mentioned it. There are none currently, there will be once #570 is resolved